twilio-authenticator-sample/src/main/java/com/twilio/authenticatorsample/appdetail/AppDetailFragment.java

```diff
@@ -43,48 +42,57 @@ public void onDismissed(Snackbar transientBottomBar, int event) {
     private AuthyTimerView authyTimerView;
     private MessageHelper messageHelper;
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_app_detail);
+    /**
+     * Use this factory method to create a new instance of Requests fragment
+     *
+     * @return A new instance of fragment RequestsFragment.
```

Comment:
I think this is not a Requests fragment, but a detail fragment.
Also, try not to use the `Request` word without the `Approval` before. We need to have consistency in our domain objects, and requests may lead to some networking stuff instead of `ApprovalRequests`

Reply:
Changed the comment in this place to say AppDetailFragment